meeshkan/gen/generator.py

```diff
@@ -157,10 +157,14 @@ def match_urls(protocol: str, host: str, o: OpenAPIObject) -> Sequence[str]:
     servers = o.servers
     if servers is None:
         return []
+    #### if the openapi server url has no scheme,
+    #### we ignore the incoming scheme and treat the url as the host
+    #### not sure if this is the right decision,
+    #### but it deals with a realistic outcome in certain schemas
     return [server.url
         for server in servers
-        if (urlparse(server.url).scheme == protocol)
-            and  (urlparse(server.url).netloc == host)]
+        if ((urlparse(server.url).scheme in ['', None]) or (urlparse(server.url).scheme == protocol))
```

Comment:
Same as `urlparse(server.url).schema in ['', None, protocol]`?

Reply:
Probably - will merge from phone to squash the bug and refactor in another branch when I get in front of a computer.